packages/synapse-react-client/src/components/ChangePassword/useChangePasswordFormState.tsx

```diff
@@ -130,6 +128,9 @@ export default function useChangePasswordFormState(
           onSubmit={(code, otpType) =>
             handleChangePasswordWithOtp(newPassword, code, otpType)
           }
+          hideReset2FA={true}
+          twoFactorAuthResetIsPending={false}
+          twoFactorAuthResetIsSuccess={false}
```

Comment:
For the case where you are changing your password using your current password, don't show an option to disable 2FA.

This UI is only shown to logged in users, who can just disable 2FA from the 2FA settings page. No need to go through the entire email flow.